### PR TITLE
doas 6.3p4 (new formula)

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/doas.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  def caveats
+    "The file at /etc/pam.d/sudo needs to be copied to /etc/pam.d/doas in order for doas to work"
+  end
+
+  test do
+    # not much else can be done without the PAM file known to be in place
+    assert_predicate bin/"doas", :exist?
+    assert_predicate bin/"vidoas", :exist?
```
Comment (carlocab) [inline Formula/doas.rb:21]: Can we try running one of these, and then checking the error message reported when it fails?

File: Formula/doas.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  desc "Execute commands as another user or root"
+  homepage "https://github.com/slicer69/doas"
+  url "https://github.com/slicer69/doas/archive/6.3p4.tar.gz"
+  version "6.3p4"
+  sha256 "e6dff62c7e38f8002ac0936f636432c52cf767f01ba703bf8723456b3c43e6de"
+  license "BSD-2-Clause"
+  head "https://github.com/slicer69/doas.git"
+
+  def install
+    system "make", "install", "DESTDIR=", "PREFIX=#{prefix}"
```
Comment (carlocab) [inline Formula/doas.rb:11]: Does adding
```ruby
cp "/etc/pam.d/sudo", pkgetc
```
Not work here?

File: Formula/doas.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  def caveats
+    "The file at /etc/pam.d/sudo needs to be copied to /etc/pam.d/doas in order for doas to work"
```
Comment (SMillerDev) [inline Formula/doas.rb:15]: This should be in the upstream documentation, it's not Homebrew specific as far as I can tell

**Comment by pthariensflame** [comment]
Sorry about the wait on this one!  I've been avoiding touching it because anxiety disorders suck.  >.<
I'll get to it soon!

**Comment by carlocab** [comment]
Don't worry about it. Take all the time that you need.

**Comment by carlocab** [comment]
Any progress here, @pthariensflame?

**Comment by pthariensflame** [comment]
Ah!  Thank you!

I'll handle this tonight.

**Pull request closed without merging**